WinUIGallery/ControlPages/DesignGuidance/GeometryPage.xaml

```diff
@@ -0,0 +1,273 @@
+                            Target="{x:Bind ShowGeometryButton3}" />
+                    </Canvas>
+                </ScrollViewer>
+
+                <ScrollViewer HorizontalScrollBarVisibility="Auto" HorizontalScrollMode="Auto">
+                    <StackPanel>
+                        <Grid Margin="0,48,0,24" HorizontalAlignment="Stretch">
+                            <Grid.ColumnDefinitions>
+                                <ColumnDefinition Width="148" />
+                                <ColumnDefinition Width="*" MaxWidth="400" />
+                                <ColumnDefinition Width="112" />
+                            </Grid.ColumnDefinitions>
+                            <TextBlock
+                                Margin="16,0,0,0"
+                                Foreground="{ThemeResource TextFillColorSecondaryBrush}"
+                                Style="{StaticResource CaptionTextBlockStyle}"
+                                Text="Corner radius" />
+                            <TextBlock
+                                Grid.Column="1"
+                                Foreground="{ThemeResource TextFillColorSecondaryBrush}"
+                                Style="{StaticResource CaptionTextBlockStyle}"
+                                Text="Usage" />
+                            <TextBlock
+                                Grid.Column="2"
+                                Foreground="{ThemeResource TextFillColorSecondaryBrush}"
+                                Style="{StaticResource CaptionTextBlockStyle}"
+                                Text="Style" />
+                        </Grid>
+                        <Grid
+                            MinHeight="68"
+                            HorizontalAlignment="Stretch"
+                            Background="{ThemeResource CardBackgroundFillColorDefaultBrush}"
+                            CornerRadius="{StaticResource OverlayCornerRadius}">
+                            <Grid.ColumnDefinitions>
+                                <ColumnDefinition Width="148" />
+                                <ColumnDefinition Width="*" MaxWidth="400" />
+                                <ColumnDefinition Width="160" />
+                                <ColumnDefinition Width="Auto" />
+                            </Grid.ColumnDefinitions>
+                            <StackPanel Orientation="Horizontal" Spacing="12">
+                                <Border
+                                    Grid.Column="1"
+                                    Width="20"
+                                    Height="20"
+                                    Margin="16,0,0,0"
+                                    Background="{ThemeResource AccentFillColorDefaultBrush}"
+                                    CornerRadius="{StaticResource OverlayCornerRadius}" />
+                                <TextBlock VerticalAlignment="Center" Text="8px" />
+                            </StackPanel>
```

Comment:
I would just use an extra column in the Grid for this. The StackPanel is kinda sticking out here.